sdk/eventhub/Azure.Messaging.EventHubs/src/EventPosition.cs

```diff
@@ -26,7 +26,7 @@ public sealed class EventPosition
         ///   which has not expired due to the retention policy.
         /// </summary>
         ///
-        public static EventPosition FirstAvailableEvent => FromOffset(StartOfStreamOffset, true);
+        public static EventPosition FirstAvailableEvent => FromOffset(StartOfStreamOffset, false);
```

Comment:
Curious the change from true to false? In the Track 1 java SDK, it is inclusive too.

Reply:
@conniey : The track one .NET code was non-inclusive;  I had it backwards.

```csharp
public static EventPosition FromStart()
{
    return EventPosition.FromOffset(StartOfStream);
}

public static EventPosition FromOffset(string offset, bool inclusive = false) { ... }
```

  [[Reference]](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.EventHubs/src/EventPosition.cs)  (lines 30-50)

Reply:
The Track 1 Java SDK is inclusive. 🤔 
https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/EventPositionImpl.java#L51

```java
    public static EventPositionImpl fromStartOfStream() {
        return new EventPositionImpl(ClientConstants.START_OF_STREAM, null, null, true);
    }

    private EventPositionImpl(String o, Long s, Instant d, Boolean i) {
        this.offset = o;
        this.sequenceNumber = s;
        this.dateTime = d;
        this.inclusiveFlag = i;
    }
```

Reply:
What is your `START_OF_STREAM` value?    .NET is using `-1`

Reply:
So are we.

Reply:
Kicking the conversation over to the Event Hubs working group to discuss across all languages.